CONTRIBUTING.md

```diff
@@ -72,3 +75,16 @@ Once your PR is approved and all CI checks have passed, do the following to merg
 ![Screenshot 4](https://i.imgur.com/Lne6GbI.png)
 * Set the merge commit message as the PR title. Additionally you can leave a description, that summarizes the whole PR, or leave it empty. But please do not send commit list with the description, because it takes too much space on git.
 ![Screenshot 5](https://i.imgur.com/ZuiGruB.png)
+
+# Opening an issue
+If it is related to a specific project, mark the title as such. For example:
+
+`Noloan: New button in PrettyButtonsActivity doesn't work`
+
+# Adding a new project
+When adding a new project:
+* Add a README.md. Mention the following:
+  * Project abstract
+  * How to install / use / run
+  * Issue and PR labelling name
```

Comment:
If it's new project, how it can have issues and PRs?
Also what is required? Label of PR/issue? What not link for example?

Reply:
Done